src/Interfaces/WiFiInterface.cpp

```diff
@@ -253,6 +253,8 @@ void WiFiInterface::print(const __FlashStringHelper *fmt, ...) {
 void *jmriClientHandler(void *arg) {
   int fd = (int)arg;
   DCCPPProtocolConsumer consumer;
+  // tell JMRI about our state:
+  consumer.feed((uint8_t*)"<s>", 3);
```

Comment:
This should be done only after the connection is fully established (after the buffer creation). There may also be a better approach than feeding the consumer as well, I'll double check.